apps/dh/api-dh/source/DataHub.WebApi.Tests/Snapshots/SchemaTests.ChangeTest.verified.graphql

```diff
@@ -1716,6 +1716,12 @@ input ChargeSortInput {
   code: SortDirection
 }
 
+input ChargeTypeSortInput {
+  name: SortEnumType
+  type: SortEnumType
+  isFee: SortEnumType
+}
```

Comment:
this doesn't look right, but I'm thinking we fix this in a later PR once we implement in-memory pagination